script.js

```diff
@@ -9,4 +10,167 @@ hamburger.addEventListener('click', () => {
+                        </div>
+                        <button class="wrapper seepopup">See project</button>
+                    </div>
+                </div>
+  
+     `;
+  }
+  worksSection.innerHTML = displayCard;
+}
+
+display();
+
+const showPopupBtn = document.querySelectorAll('.seepopup');
+const popUpContainer = document.createElement('section');
+popUpContainer.className = 'popup_section';
+popUpContainer.classList.add('bg_color6');
+showPopupBtn.forEach((e, i) => {
+  e.addEventListener('click', () => {
+    const popupEle = `
+    <article class="popup_article">
+
+        <button type="button" class="close_popup">&times;</button>
+    
+        <img class="bg-popup" src="images/Snapshoot Portfolio.jpg" alt="">
+        <img class="bg-popup-m" src="images/SPMobile.jpg" alt="">
+        <div class="mod-div-1">
+            <div class="title-popup">
+                <h4>Keeping track of hundreds of components</h4>
+                <div class="mod-div-2-1">
+                    <button  class="wrapper" id="see-projects-btn" type="submit"><a href="${cardsData[i].liveLink}"><span>See Live</span><img class="icn" src="images/Icon-Export.png" alt="see-live">
+                    </a></button>
+                    <button class="wrapper" id="see-projects-btn" type="submit"><a href="${cardsData[i].Source}"><span>See Source</span><img class="icn" src="images/Icon-GitHub.png" alt="see-source">
+                    </a></button>
+                </div>
+            </div>
+            <div class="desln">
+                <ul class="desln">
+                    <li><a class="lang-popup" href="#">Codekit</a></li>
+                    <li><a class="lang-popup" href="#">GitHub</a></li>
+                    <li><a class="lang-popup" href="#">Javascript</a></li>
+                    <li><a class="lang-popup" href="#">Bootstrap</a></li>
+                    <li><a class="lang-popup" href="#">Terminal</a></li>
+                    <li><a class="lang-popup" href="#">Codepin</a></li>
+                </ul>
+            </div>
+            <div class="mobln">
+                <ul>
+                    <li><a class="lang-popup" href="#">Ruby on Rails</a></li>
+                    <li><a class="lang-popup" href="#">CSS</a></li>
+                    <li><a class="lang-popup" href="#">Javascript</a></li>
+                </ul>
+            </div>
+        </div>
+
+        <div class="mod-div-3"><p class="details">${cardsData[i].description}</p></div>
+        <div class="mod-div-2">
+            <button  class="wrapper" id="see-projects-btn" type="submit"><a href="${cardsData[i].liveLink}"><span>See Live</span><img class="icn" src="images/Icon-Export.png" alt="see-live">
+            </a></button>
+            <button class="wrapper" id="see-projects-btn" type="submit"><a href="${cardsData[i].Source}"><span>See Source</span><img class="icn" src="images/Icon-GitHub.png" alt="see-source">
+            </a></button>
+        </div>
+        
+    </article>  
```

Comment:
- You have done a good job so far, but you have not properly designed the popup buttons. Remember it is important that you try to replicate the Figma design as closely as possible to give confidence to recruiters in your ability to produce exact business requirements 🙏
Desktop version:
- Please note that the icons should be on the same line as the texts, I suggest giving your popup a little bit more width to accommodate the changes.
![6ue46ue4](https://github.com/zhorabay/My-portfolio/assets/73607512/b5d9bb5f-a576-40c2-ae22-08dc5726d8d1)
Your project(desktop version):
![rrrjdryjdrj](https://github.com/zhorabay/My-portfolio/assets/73607512/e6b9b603-b535-4e63-8b7f-627a0bab98bc)

Mobile version(Figma):
- Please note that the buttons occupy the entire width of the popup with space between them. Please fix this 👍 
![jryjdryj](https://github.com/zhorabay/My-portfolio/assets/73607512/0c60c632-c81b-4e05-b46d-eda110d3d00a)
Mobile version(Your project):
![yrjeryj](https://github.com/zhorabay/My-portfolio/assets/73607512/18803ccd-cdc1-4773-90c6-0ed6583d57be)